### DISCUSSION TITLE
Sozi release 22.09 for testers

### DISCUSSION THREAD

**Discussion by aumouvantsillage** [senshu/Sozi#656] [Announcements]
This is mainly a bugfix release. It addresses the following issues:
* Fixed PDF export (#645)
* Unexpected transformations when the document contains embedded SVGs (#646 #647)

This release also contains updated translations.

Under the hood, it is based on [Electron](https://www.electronjs.org/) 20 and brings an updated version of the SVG rendering engine.
While Sozi itself has not changed a lot since the previous release, updating Electron has required a few adaptations internally, which could result in unexpected side effects.
For this reason, this release is published for testing and will hopefully be promoted as the new stable release soon.

Installation instructions can be found in the Sozi guide: [in English](http://sozi.guide/en/installation.html), [en français](http://sozi.guide/fr/installation.html).

[![Buy me a Coffee](https://www.buymeacoffee.com/assets/img/custom_images/orange_img.png)](https://www.buymeacoffee.com/THtbNvnqE)

<hr /><em>This discussion was created from the release <a href='https://github.com/sozi-projects/Sozi/releases/tag/v22.09-beta'>Sozi release 22.09 for testers</a>.</em>

**Comment by raniaamina** [comment]
Thanks for the update!
